lib/rubocop/cop/style/class_and_module_children.rb

```diff
@@ -125,14 +125,19 @@ def remove_end(corrector, body)
           corrector.remove(range)
         end
 
+        # rubocop:disable Metrics/AbcSize
         def unindent(corrector, node)
           return if node.body.children.last.nil?
```

Comment:
To avoid `Metrics/AbcSize`, replacing it with `unless` could be considered.

```suggestion
          return unless node.body.children.last
```